css/media.css

```diff
@@ -0,0 +1,1201 @@
+@media only screen and (max-width: 1024px) {
+	.header-content ,
+	.poster-content ,
+	.product-content {
+		max-width: 1024px;
+	}
+
+	.header-container ,
+	.poster-container ,
+	.product-container ,
+	.product-items {
+		max-width: 930px;
+	}
+
+	.poster-container{
+		height: 252px;
+	}
+		
+	.product {
+		margin-bottom: 75px;
+	}
+		
+	.product-tile {
+		max-width: 208px;
+	}
+
+	.product-tile__image {
+		max-width: 169px;
+	}
+
+	.product-tile:not(:last-child):after {
+		right: -17px;
+	}
+
+/*info start*/
+	.info {
+		max-width: 1024px;
+		margin-bottom: 93px;
+	}
+
+	.info-content {
+		max-width: 930px;
+		height: 442px;
+	}
+	
+	.info-container {
+		max-width: 854px;
+		margin: 0 38px;
+	}
+
+	.info__item {
+    width: 55vw;
+    height: 35vh;
+		margin-right: 46px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Project_Axion/pullRequest?prid=5457134)